tests/perf/service_invocation_http/service_invocation_http_test.go

```diff
@@ -183,5 +183,5 @@ func TestServiceInvocationHTTPPerformance(t *testing.T) {
 	require.Equal(t, 0, restarts)
 	require.True(t, daprResult.ActualQPS > float64(p.QPS)*0.99)
 	require.Greater(t, tp90Latency, 0.0)
-	require.LessOrEqual(t, tp90Latency, 3.0)
+	require.LessOrEqual(t, tp90Latency, 5.0)
```

Comment:
This isn't needed. I've observed other tests to be flaky, or flakier than others, but haven't seen issues with the HTTP invocation test. Can you provide test run evidence for this?

Reply:
This is the latest records I collected from github perf test case: 

```bash
# https://github.com/dapr/dapr/runs/6901427636?check_suite_focus=true
service_invocation_http_test.go:141: target dapr sidecar consumed 58m Cpu and 43.216896Mb of Memory
service_invocation_http_test.go:162: added latency for 50th percentile: 1.11ms
service_invocation_http_test.go:162: added latency for 75th percentile: 1.67ms
service_invocation_http_test.go:162: added latency for 90th percentile: 1.68ms
service_invocation_http_test.go:162: added latency for 99th percentile: 2.80ms
service_invocation_http_test.go:165: baseline latency avg: 0.78ms
service_invocation_http_test.go:166: dapr latency avg: 1.88ms
service_invocation_http_test.go:167: added latency avg: 1.10ms


# https://github.com/dapr/dapr/runs/6903629684?check_suite_focus=true
service_invocation_http_test.go:141: target dapr sidecar consumed 2m Cpu and 32.428032Mb of Memory
service_invocation_http_test.go:162: added latency for 50th percentile: 1.75ms
service_invocation_http_test.go:162: added latency for 75th percentile: 1.41ms
service_invocation_http_test.go:162: added latency for 90th percentile: 2.77ms
service_invocation_http_test.go:162: added latency for 99th percentile: 3.72ms
service_invocation_http_test.go:165: baseline latency avg: 0.92ms
service_invocation_http_test.go:166: dapr latency avg: 2.29ms
service_invocation_http_test.go:167: added latency avg: 1.37ms

# https://github.com/dapr/dapr/runs/6903718080?check_suite_focus=true
service_invocation_http_test.go:141: target dapr sidecar consumed 238m Cpu and 43.057151999999995Mb of Memory
service_invocation_http_test.go:162: added latency for 50th percentile: 1.45ms
service_invocation_http_test.go:162: added latency for 75th percentile: 1.21ms
service_invocation_http_test.go:162: added latency for 90th percentile: 2.24ms
service_invocation_http_test.go:162: added latency for 99th percentile: 2.16ms
service_invocation_http_test.go:165: baseline latency avg: 0.98ms
service_invocation_http_test.go:166: dapr latency avg: 2.20ms
service_invocation_http_test.go:167: added latency avg: 1.22ms
```

3 ms is OK for service invoke http now, but you can see that the max value of 90th is 2.77 ms. 

There are some records I collected last week (see https://github.com/dapr/dapr/issues/4743#issuecomment-1152504421) :

```bash
service_invocation_http_test.go:158: added latency for 75th percentile: 0.99ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.05ms

service_invocation_http_test.go:158: added latency for 75th percentile: 1.77ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.37ms

service_invocation_http_test.go:158: added latency for 75th percentile: 1.50ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.83ms

service_invocation_http_test.go:158: added latency for 75th percentile: 1.01ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.35ms

service_invocation_http_test.go:158: added latency for 75th percentile: 1.71ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.78ms

service_invocation_http_test.go:158: added latency for 75th percentile: 1.58ms
service_invocation_http_test.go:158: added latency for 90th percentile: 1.86ms
```

The performance is coming slower and slower.

I can rollback this change, but since service invoke http is the first test case of all, if it failed, then we don't have changes to run other test case.

Reply:
I rollback it, back to use 3 ms. (The original value is 2 ms)

Reply:
Up until this test run https://github.com/dapr/dapr/runs/6895595104?check_suite_focus=true all results were consistent and I see since then the perf tests have changed output. Can you point me to the change in tests that occured?